inbound_test.go

```diff
@@ -142,3 +142,27 @@ func TestInboundConnection_CallOptions(t *testing.T) {
 		require.NoError(t, err, "Call through proxy failed")
 	})
 }
+
+func TestBlackhole(t *testing.T) {
+	testutils.WithTestServer(t, nil, func(tServer *testutils.TestServer) {
+		serviceName := tServer.ServiceName()
+
+		server := tServer.NewServer(testutils.NewOpts())
```

Comment:
why are you creating a new server here? the `tServer` can have methods registered directly

Reply:
yup, my bad. I overlooked the `register` function completely.